orderbook/src/orderbook.rs

```diff
@@ -126,7 +126,7 @@ impl Orderbook {
             _ => (),
         }
         self.balance_fetcher
-            .register(order.order_creation.sell_token, order.order_meta_data.owner)
+            .register(order.order_meta_data.owner, order.order_creation.sell_token)
```

Comment:
This is such a classic error (same type, unnamed arguments) 🙈. Maybe we should create a named struct instead to make sure this doesn't happen again in the future

Reply:
Personally I don't think it's needed but I'm not very against it either.

Reply:
So... how were we ever getting correct information somtimes?